hera_sim/simulate.py

```diff
@@ -57,26 +57,31 @@ def __init__(self, data=None, defaults_config=None, **kwargs):
         # actually initialize the UVData object stored in self.data
         self._initialize_data(data, **kwargs)
 
-    @cached_property
+    @property
     def antpos(self):
         # TODO: docstring
```

Comment:
TODO

Reply:
Planning on leaving this for a PR dedicated to updating documentation, unless you're strongly against this.

Reply:
Yup, that's fine.